### PR TITLE
contrib/podman-tui: new package (1.2.0)

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

File: contrib/podman-tui/template.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pkgver = "1.2.0"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+makedepends = ["linux-headers", "libbtrfs-devel"]
+checkdepends = ["pkgconf", "gpgme-devel"]
+go_build_tags = [
+    "exclude_graphdriver_devicemapper",
+    "exclude_graphdriver_btrfs",
+    "btrfs_noversion",
```
Comment (nekopsykose) [inline contrib/podman-tui/template.py:11]: these are definitely not meant to be set

Comment (nekopsykose) [reply contrib/podman-tui/template.py:11]: hm well they're set by default, though in alpine they're not..

Comment (nekopsykose) [reply contrib/podman-tui/template.py:11]: let's assume it works

Comment (ttyyls) [reply contrib/podman-tui/template.py:17]: i got them from the project's makefile.

**Pull request merged**